setup.py

```diff
@@ -0,0 +1,37 @@
+from setuptools import setup, find_packages
+import os
+
+
+script_dir = 'src/emcpy/bin'
+scripts = [os.path.join(script_dir, x) for x in os.listdir(script_dir)]
+
+setup(
+    name='emcpy',
+    author='NOAA-EMC',
+    version='0.0.1',
+    description='A collection of python tools used at EMC',
+    url='https://github.com/noaa-emc/emcpy',
+    keywords='NOAA, EMC',
+    package_dir={'': 'src'},
+    packages=find_packages(where='src'),
+    package_data={
+        '': ['cfg/*'],
+    },
+    include_package_data=True,
+    scripts=scripts,
+    python_requires='>= 3.6',
+    install_requires=[
+        'pyyaml',
```

Comment:
We will need to change this soon, but fine for now?

Reply:
what specifically will w need to change? `install_requires`?  I am leaning towards creating a `setup.cfg` file to include metadata so we don't have to touch `setup.py`

Reply:
will need cartopy, matplotlib, yada yada

Reply:
yeah sure.  As we add dependencies, we should update that list here.